package.json

```diff
@@ -28,6 +28,17 @@
     "engines": {
         "node": ">=4.2.0"
     },
+    "files": [
+        "./bin",
```

Comment:
I made these all `./`, but that's not actually required I think.

Reply:
(But, I'll probably leave it that way, as it mirrors the other paths we use in the file; see above.)